Visual Studio 2015 Windows XP (v140_xp) Platform Toolset

Master and release-1.10.0 CMake builds fail when the Platform Toolset in Visual Studio 2015 is set to Windows XP (v140_xp). It is unable to find header **debugapi.h**
Tag release-1.8.1 does not have this issue. And the standard Platform Toolset (v140) does not have this issue.

```
C:\git\googletest-release-1.10.0\_build>cmake -G "Visual Studio 14 2015" -A "Win32" -T "v140_xp" ..\
-- Selecting Windows SDK version  to target Windows 10.0.17763.
-- The C compiler identification is MSVC 19.0.24215.1
-- The CXX compiler identification is MSVC 19.0.24215.1
-- Check for working C compiler: C:/Program Files (x86)/Microsoft Visual Studio 14.0/VC/bin/cl.exe
-- Check for working C compiler: C:/Program Files (x86)/Microsoft Visual Studio 14.0/VC/bin/cl.exe -- works
-- Detecting C compiler ABI info
-- Detecting C compiler ABI info - done
-- Detecting C compile features
-- Detecting C compile features - done
-- Check for working CXX compiler: C:/Program Files (x86)/Microsoft Visual Studio 14.0/VC/bin/cl.exe
-- Check for working CXX compiler: C:/Program Files (x86)/Microsoft Visual Studio 14.0/VC/bin/cl.exe -- works
-- Detecting CXX compiler ABI info
-- Detecting CXX compiler ABI info - done
-- Detecting CXX compile features
-- Detecting CXX compile features - done
-- Found PythonInterp: C:/Program Files (x86)/Python38-32/python.exe (found version "3.8.1")
-- Looking for pthread.h
-- Looking for pthread.h - not found
-- Found Threads: TRUE
-- Configuring done
-- Generating done
-- Build files have been written to: C:/git/googletest-release-1.10.0/_build

C:\git\googletest-release-1.10.0\_build>cmake --build . --config Debug
Microsoft (R) Build Engine version 14.0.25420.1
Copyright (C) Microsoft Corporation. All rights reserved.

  Checking Build System
  Building Custom Rule C:/git/googletest-release-1.10.0/googlemock/CMakeLists.txt
  gtest-all.cc
C:\git\googletest-release-1.10.0\googletest\src/gtest.cc(87): fatal error C1083: Cannot open include file: 'debugapi.h': No such file or directory [C:\git\googletest-release-1.10.0\_build\googlemock\gmock.vcxproj]
  gmock-all.cc
  Generating Code...
  Building Custom Rule C:/git/googletest-release-1.10.0/googlemock/CMakeLists.txt
  gtest-all.cc
C:\git\googletest-release-1.10.0\googletest\src/gtest.cc(87): fatal error C1083: Cannot open include file: 'debugapi.h': No such file or directory [C:\git\googletest-release-1.10.0\_build\googlemock\gmock_main.vcxproj]
  gmock-all.cc
  gmock_main.cc
  Generating Code...
  Building Custom Rule C:/git/googletest-release-1.10.0/googletest/CMakeLists.txt
  gtest-all.cc
C:\git\googletest-release-1.10.0\googletest\src/gtest.cc(87): fatal error C1083: Cannot open include file: 'debugapi.h': No such file or directory [C:\git\googletest-release-1.10.0\_build\googletest\gtest.vcxproj]
```

--- a/googletest/src/gtest.cc
+++ b/googletest/src/gtest.cc
@@ -86,7 +86,6 @@
 
 #ifdef _MSC_VER
 # include <crtdbg.h>  // NOLINT
-# include <debugapi.h>  // NOLINT
 #endif
 
 # include <io.h>  // NOLINT

--- a/googletest/src/gtest.cc
+++ b/googletest/src/gtest.cc
@@ -81,7 +81,6 @@
 
 #elif GTEST_OS_WINDOWS  // We are on Windows proper.
 
-# include <Windows.h>  // NOLINT
 # include <windows.h>  // NOLINT
 # undef min
 

